llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -5430,6 +5430,13 @@ Node *AbstractManglingParser<Derived, Alloc>::parseEncoding() {
   if (IsEndOfEncoding())
     return Name;
 
+  // ParseParams may be false in top-level only, when called from parse().
```

Comment:
I think this comment would be enhanced by an example.

Reply:
Not sure I follow. Could you elaborate, please? What comment enhancement by an example you would like to see here?

Reply:
It's mostly to completely clarify what is meant by "top-level" here. For an example, I'd take a mangled name that contains a nested function signature and show which part is considered top-level and which part isn't. E.g.
```
  // ParseParams maybe false at the top level only, when called from parse(). For example in
  // the mangled name <mangled name>, ParseParams may be true when demangling
  // <top-level part> but is always false when demangling <nested part>.
```
(obviously replace the bits between `<>` with appropriate strings and reflow as appropriate).